app/models/ability.rb

```diff
@@ -3,7 +3,10 @@ class Ability
 
   def initialize(user)
     if user.class == Admin
-      can :read, Admin, id: user.id
+      can [:read, :update, :destroy], Admin, id: user.id
+      can [:new, :create, :destroy], Admin
+      can [:read, :edit, :update, :new, :create, :destroy], Item
+      can [:edit, :update, :new, :create, :destroy], Category
     elsif user.class == User
```

Comment:
Actuall I believe the correct syntax is can :manage Item, and can :manage Category